data/field_config.go

```diff
@@ -148,10 +148,9 @@ const (
 
 // ValueMapping convert input value to something else
 type ValueMapping struct {
-	ID       int16       `json:"id"`
-	Operator string      `json:"operator"`
-	Text     string      `json:"title"`
-	Type     MappingType `json:"type"`
+	ID   int16       `json:"id,omitempty"`
+	Text string      `json:"text,omitempty"`
```

Comment:
Just making sure, the JSON field should be "text" and not "title" (as before)?

Reply:
yup -- that is the `bug` part of this PR :)